### DISCUSSION TITLE
Take album art from Last.FM?

### DISCUSSION THREAD

**Discussion by n0stal6ic** [s0hv/foo_discord_rich#27] [Ideas]
Quick idea for the project, I think it would be very beneficial to take the cover art from Last.FM instead of uploading it to Imgur.
This way it will never fail and you can upload your own custom covers to the service if it does not have one.
People usually love to upload images in high-quality to last.fm and I know that the majority of people who use foobar2000 also use the last.fm component as well.

Not saying the Imgur uploader has to be totally removed, just the option to not use it. This also gives good GIF support from last.fm.
I, personally, would not have to worry about if my uploads fail to go through on imgur or if I get rate-limited. Would also save bandwidth.

You can grab images directly from the album and change the url scheme to remove the resolution section to make it the biggest one.

**Comment by fitsfer** [comment]
the OG project already has an option to pull from MusicBrainz...

**Reply by n0stal6ic** [reply]
Would it be appropriate to make a discussion on the og project on this?
Sometimes the MusicBrainz one is not accurate to the album I have.
I know for sure community sourced ones on last.fm are usually always the best.

**Reply by s0hv** [reply]
I would say that move the discussion there. I'm not sure what the future plans are for this fork, as the original repository should support my use case and more.

**Reply by n0stal6ic** [reply]
Yeah, sorry I haven't been checking the original repo because I would always use your fork. Didn't know it was still updated. 
Thank you for the reply though! Will move it over.
